app/pyalbert_utils.py

```diff
@@ -91,7 +91,7 @@ def generate(config: Config, query: str) -> str | None:
     return answer
 
 
-def generate_sources(config: Config, stream_id: int) -> list[dict | None]:
```

Comment:
@dtrckd Same. This raises a Mypy error if it doesn't take into account the None case.

Reply:
Because the list is a list of dict, it can't be None ?

Reply:
The returned list can be empty: https://github.com/etalab-ia/albert-tchapbot/blob/dev/app/pyalbert_utils.py#L108

Reply:
Can I revert your commit?